tz.cpp

```diff
@@ -410,23 +410,25 @@ load_timezone_mappings_from_csv_file(const std::string& input_path)
     for (int i = 0; i < 4; ++i)
         getline(is, copyright);
     
```

Comment:
Okay.. the changes to `load_timezone_mappings_from_csv_file` are not necessary, I should have left them out, as well as the changes to `timezone_mapping`